### PR TITLE
test/run: shellcheck fixes

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

**Comment by junegunn** [comment]
Thanks, how about if we just `set -eu`? So that things like `|| exit` are not needed.

**Comment by blueyed** [comment]
Amended.

File: test/run [hunk 1]
```diff
@@ -39,13 +43,13 @@ clone_repos() (
 )
 
 make_dirs() (
-  rm -rf "$PLUG_FIXTURES/$1"
+  rm -rf "${PLUG_FIXTURES:?}/$1"
```
Comment (junegunn) [inline test/run]: I believe this is also covered by `set -u`.

**Comment by junegunn** [comment]
Sorry about the delay. Thanks!

**Pull request merged**